src/classes/OPP_SendAcknowledgmentBTN_CTRL.cls

```diff
@@ -119,7 +119,7 @@ public with sharing class OPP_SendAcknowledgmentBTN_CTRL {
 
         } else {
             oppIds.addAll(
-                extractIds(setController.getSelected())
+                UTIL_SObject.extractIds(setController.getSelected())
```

Comment:
@heidmary , the method has been replaced with a common utility method, Behaviour should be unchanged. PR #3897 contains more detail on how to test.